plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -29,6 +30,7 @@
 
 import java.util.Map;
 
+import static io.airlift.log.Level.OFF;
```

Comment:
Should we be consistent about static importing `Level` values? I'm fine either way, but would prefer consistency.